### PR TITLE
4739 excessive use of font weight bold throughout the UI

### PR DESCRIPTION
**Pull request by robertdown**
Removing excess font-weight-bold, slowly.

### CODE REVIEW

File: interface/forms/fee_sheet/new.php [hunk 1]
```diff
@@ -308,13 +308,13 @@ function echoServiceLines()
...
-            echo "<input type='text' class='form-control' name='bill[" . attr($lino) . "][ndcnum]' value='" . attr($li['ndcnum']) . "' " .
+            echo "<input type='text' class='form-control form-control-sm' name='bill[" . attr($lino) . "][ndcnum]' value='" . attr($li['ndcnum']) . "' " .
             "size='11' />";
             echo " &nbsp;Qty:&nbsp;";
-            echo "<input type='text' class='form-control text-left' name='bill[" . attr($lino) . "][ndcqty]' value='" . attr($li['ndcqty']) . "' " .
+            echo "<input type='text' class='form-control form-control-sm text-left' name='bill[" . attr($lino) . "][ndcqty]' value='" . attr($li['ndcqty']) . "' " .
             "size='3' />";
             echo " ";
-            echo "<select class='form-control' name='bill[" . attr($lino) . "][ndcuom]'>";
+            echo "<select class='form-control form-sontrol-sm' name='bill[" . attr($lino) . "][ndcuom]'>";
```
Comment (stephenwaite) [inline interface/forms/fee_sheet/new.php:317]: `form-control-sm`

File: interface/forms/fee_sheet/new.php [hunk 2]
```diff
@@ -1200,40 +1200,42 @@ function defaultPriceLevelChanged(sel) {
...
+                                            <th class='billcell'><?php echo xlt('Revenue');?></th>
                                         <?php } ?>
-                                        <!-- Price display is conditional. -->
-                                        <?php if ($fs->pricesAuthorized()) { ?>
-                                        <td class='billcell text-right font-weight-bold'><?php echo xlt('Price');?>&nbsp;</td>
-                                        <?php } else { ?>
-                                        <td class='billcell' style='display:none'>&nbsp;</td>
+                                        <?php if (modifiers_are_used(true)) { ?>
+                                            <th class='billcell'><?php echo xlt('Modifiers');?></th>
                                         <?php } ?>
```
Comment (stephenwaite) [inline interface/forms/fee_sheet/new.php:1213]: bradys-panda

**Review by stephenwaite — APPROVED** [review]

File: interface/drugs/drug_inventory.php [hunk 3]
```diff
@@ -382,50 +382,24 @@ function facchanged() {
...
+            <th><?php echo xlt('NDC'); ?> </a></th>
+            <th><?php echo xlt('Form'); ?> </a></th>
+            <th><?php echo xlt('Size'); ?></th>
+            <th title='<?php echo xlt('Measurement Units'); ?>'><?php echo xlt('Unit'); ?></th>
+            <th title='<?php echo xla('Purchase or Transfer'); ?>'><?php echo xlt('Tran'); ?></th>
+            <th><?php echo xlt('Lot'); ?> </a></th>
+            <th><?php echo xlt('Facility'); ?> </a></th>
+            <th><?php echo xlt('Warehouse'); ?> </a></th>
+            <th><?php echo xlt('QOH'); ?> </a></th>
+            <th><?php echo xlt('Expires'); ?> </a></th>
```
Comment (bradymiller) [inline interface/drugs/drug_inventory.php:400]: looks like this was also in the original code, but why are there a bunch of lone `</a>` tags in above code?

File: interface/drugs/drug_inventory.php [hunk 4]
```diff
@@ -382,50 +382,24 @@ function facchanged() {
...
+<table id='mymaintable' class="table table-striped">
+    <thead>
+        <tr>
+            <th><?php echo xlt('Name'); ?> </a></th>
+            <th><?php echo xlt('Act'); ?></th>
+            <th><?php echo xlt('Cons'); ?></th>
+            <th><?php echo xlt('NDC'); ?> </a></th>
+            <th><?php echo xlt('Form'); ?> </a></th>
+            <th><?php echo xlt('Size'); ?></th>
+            <th title='<?php echo xlt('Measurement Units'); ?>'><?php echo xlt('Unit'); ?></th>
```
Comment (bradymiller) [inline interface/drugs/drug_inventory.php:394]: also wrong in original code, but should be using xla for the above title= string

**Comment by bradymiller** [comment]
2DV-2

(btw, noted couple issues that not your fault (ie. in old code), which will plan to fix up in future)

**Pull request merged**